tool/tsh/common/kube_proxy.go

```diff
@@ -198,7 +198,7 @@ func makeKubeLocalProxy(cf *CLIConf, tc *client.TeleportClient, clusters kubecon
 		return nil, trace.Wrap(err)
 	}
 
-	localClientKey, err := keys.LoadPrivateKey(profile.KeyPath())
+	localClientKey, err := native.GeneratePrivateKey()
```

Comment:
Just leave a comment saying why we need to generate a private key.

Before we derived the local CA from the user's private key to improve initialization speed.